src/test/scala/com/agar/core/gameplay/player/PlayerSpec.scala

```diff
@@ -22,42 +23,37 @@ class PlayerSpec (_system: ActorSystem)
     "wander when he has nothing around" in {
       implicit val agarSystem: AgarSystem = () => 1 second
 
-      val testProbe = TestProbe()
-      val tracer = system.actorOf(Props(new Tracer(testProbe.ref)))
-
       val playerPosition = Vector2d(100, 100)
-      val playerRef = system.actorOf(Props(new Player(playerPosition, 10)(tracer)))
+      val playerRef = system.actorOf(Props(new Player(playerPosition, 10)))
 
       val aoi = AOI(List.empty, List.empty)
 
       playerRef ! Tick(aoi)
-      testProbe.expectMsgPF() {
-        case MovePlayer(ref, position) => playerRef == ref && position != playerPosition
+      this.expectMsgPF() {
+        case MovePlayer(position, _) => position != playerPosition
```

Comment:
Removed the `ref == playerRef` make the test less strong no ?

Reply:
The reference has been removed. We can check the `lastSender` right now otherwise.